pipe_segment/segment_identity/transforms.py

```diff
@@ -39,13 +39,15 @@ def extract(identities, key):
             mapping[value] += ident["count"]
     return [{"value": k, "count": v} for (k, v) in mapping.items()]
 
-
 def summarize_identifiers(segment):
     identities = segment["daily_identities"]
+    counts = [ident["count"] for ident in identities]
     transponders = extract(identities, "transponder_type")
     shipnames = extract(identities, "shipname")
     callsigns = extract(identities, "callsign")
     imos = extract(identities, "imo")
+    lengths = extract(identities, "length")
+    widths = extract(identities, "width")
 
```

Comment:
This might be clearer if you compute msg , ident and pos counts here then sub them in down below. In particular, it would be easy to make clear that msg_count is the sum of pos and ident counts. Something like ```
pos_count = segment.get("daily_msg_count")
ident_count = sum(counts)
msg_count = pos_count + ident_count  # Or could do inline down below.
```

Reply:
I like that, thanks! Just made that change and pushed to this branch. Should be reflected in PR now